src/app/modules/form/admin-form/admin-form.middlewares.ts

```diff
@@ -26,5 +27,23 @@ export const updateSettingsValidator = celebrate({
       url: Joi.string().uri().allow(''),
       isRetryEnabled: Joi.boolean(),
     }).min(1),
-  }).min(1),
+  })
+    .min(1)
+    .custom((value, helpers) => {
+      // If there are unicode-escaped characters are not valid utf-8 encoded,
+      // node 14 treats the sequence of characters as a string e.g. \udbbb is treated as a 6-character string instead of an escaped unicode sequence
+      // If this is saved into the db, an error is thrown when the driver attempts to read the db document as the driver interprets this as an escaped unicode sequence
+      // Since valid unicode-escaped characters will be processed correctly (e.g. \u00ae is processed as ®), they will not trigger an error
+      // Also note that if the user intends to input a 6-character string of the same form e.g. \udbbb, the backslash will be escaped (i.e. double backslash) and hence this will also not trigger an error
+
+      const valueStr = JSON.stringify(value)
+
+      if (UNICODE_ESCAPED_REGEX.test(valueStr)) {
+        return helpers.message({
+          custom:
+            'Please check that there are no improperly encoded characters in your input',
```

Comment:
I wonder if folks would understand this error message 🤔 . Since we **know** there are invalid UTF8 characters in the input, we should probably say it outright.
```suggestion
            'There are invalid characters in your input',
```